### DISCUSSION TITLE
STELLA-Q2 Build

### DISCUSSION THREAD

**Discussion by DoubleOut** [STELLA-Landsat/STELLA#65] [Show and tell]
My first build of the Q2!

![image](https://github.com/STELLA-Landsat/STELLA/assets/3082644/6cee5e60-4174-4196-bcfd-45ee593a96e2)

![image](https://github.com/STELLA-Landsat/STELLA/assets/3082644/d1bff63e-45dc-4877-a386-4cd95724837f)

**Comment by captoriginal** [comment]
The mag-safe usb-c connector is a nice touch!

**Comment by paulmirel** [comment]
It looks great! 

Yes, the MagSafe connector is great! Where'd you get it? 

P

**Reply by DoubleOut** [reply]
I picked it up on Amazon, there's a ton to choose from.
